tools/rebasehelpers/util/git.go

```diff
@@ -343,7 +343,7 @@ func filesInCommit(sha string) ([]File, error) {
 
 func descriptionInCommit(sha string) ([]string, error) {
 	descriptionLines := []string{}
-	stdout, stderr, err := run("git", "show", "--quiet", sha)
+	stdout, stderr, err := run("git", "log", "--pretty=%b", "-1", sha)
```

Comment:
The difference in that snippet being the two trailing newlines at the end of the `git show` outupt